packages/manager/apps/dedicated/client/app/billing/autoRenew/actions/terminateEmail/email.routing.js

```diff
@@ -25,8 +24,18 @@ export default /* @ngInject */ ($stateProvider) => {
       name: /* @ngInject */ ($transition$) => $transition$.params().name,
       serviceId: /* @ngInject */ ($transition$) =>
         $transition$.params().serviceId,
-      terminateEmail: /* @ngInject */ (BillingAutoRenew, email) => () =>
-        BillingAutoRenew.terminateEmail(email),
+      terminateEmail: /* @ngInject */ (BillingAutoRenew, serviceId) => () =>
+        BillingAutoRenew.terminateEmail(serviceId),
     },
+  };
+
+  $stateProvider.state(
+    'app.account.billing.autorenew.terminateEmail',
+    stateDefinition,
+  );
+
+  $stateProvider.state('app.account.billing.autorenew.terminateEmailDomain', {
+    ...stateDefinition,
+    url: '/delete-email-domain?serviceId&name',
```

Comment:
Why create a duplicate state instead of using a $urRouterProvider.when redirection ? This is quite confusing, which state should be called in case of a `$state.href` ?